.env

```
@@ -1,4 +1,6 @@
 PORT=8080
+DB_URL=postgresql://neondb_owner:npg_ytmANVb94jKY@ep-soft-field-a164h2d0-pooler.ap-southeast-1.aws.neon.tech/neondb?sslmode=require&channel_binding=require
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_URL key should go before the PORT key

(UnorderedKey)